res/openvpn-gui-res.rc

```diff
@@ -58,6 +59,9 @@ ID_ICO_DISCONNECTED  ICON  DISCARDABLE  "disconnected.ico"
 #include "openvpn-gui-res-ua.rc"
 #include "openvpn-gui-res-zh-hans.rc"
 #include "openvpn-gui-res-zh-hant.rc"
+#else
+#include "openvpn-gui-res-en-msvc.rc"
+#endif
```

Comment:
I think this would be cleaner if used as
```
#ifdef _MSC_VER
#include "openvpn-gui-res-en-msvc.rc"
#else 
<existing includes>
#endif 
```

Reply:
agreed